### DISCUSSION TITLE
Recording external mic and system audio into Adobe Audition

### DISCUSSION THREAD

**Discussion by tonywatkins** [ExistentialAudio/BlackHole#395] [General]
I've spent several hours fiddling around with various combinations of things to crack the problem of capturing system audio and external microphone for recording lectures and webinars I give via Zoom etc. using Adobe Audition. I I find it amazing that there is not much more advice around on setting this up. I was very pleased to come across BlackHole earlier this morning – I always used to use Soundflower but I can't get it to do what I want with Big Sur (OS 11.2.2). The critical piece of information was in the BlackHole wiki – the built-in output must be the top device in the Multi-output device. I've not seen this stated anywhere else. It still took some experimentation, but I've finally got there. Here's what I did:

1. Set up multi-output device with MacBook Pro Speakers (listed as Built-in Output for some people) first in the list, plus my headphones and BlackHole 2ch (and right-click on this to select 'use this device for sound output'):

![Screenshot 2021-05-20 at 11 57 50](https://user-images.githubusercontent.com/25526985/118967810-0ca56980-b963-11eb-99c8-7ac4a783656a.png)

2. Set up aggregate device with external mic first and BlackHole 2ch (and right-click to select 'use this device for sound input'):

![Screenshot 2021-05-20 at 11 57 14](https://user-images.githubusercontent.com/25526985/118967754-feefe400-b962-11eb-9259-2bcaa37cc9ee.png)

3. in Audition preferences, set the audio hardware default input to aggregate device:

![Screenshot 2021-05-20 at 12 03 15](https://user-images.githubusercontent.com/25526985/118968141-77ef3b80-b963-11eb-9925-f22e41b3b96b.png)

4. In Audition preferences, set one channel to Aggregate device: External microphone and the other to Aggregate Device: BlackHole 2ch: BlackHole 2 ch 1:

![Screenshot 2021-05-20 at 12 04 21](https://user-images.githubusercontent.com/25526985/118968326-af5de800-b963-11eb-9ead-8ee28339bbaa.png)

What I haven't yet worked out is if I can use both channels from BlackHole plus the external mic – maybe if I'm using multi-channel recording in Audition, but for what I'm doing it's not a problem. Hope this helps others who are struggling with this!

**Comment by SegoviaSIFT** [comment]
I am struggling to set up Internal Audio and External Mic into Quicktime, I tried your solution but it didn't work. Did you do any more investigations?
